pyproject.toml

```diff
@@ -15,6 +15,9 @@ addopts= """
 """
 log_file_level="INFO"
 
+[tool.pylint]
+load-plugins="pylint.extensions.no_self_use"
```

Comment:
Pylint got updated, the no_self_use option is now opt in so this turns that back on